docs/docker_walkthrough.rst

```diff
@@ -0,0 +1,539 @@
+
+The next service we are adding is Rasa NLU:
+
+.. code-block:: yaml
+
+  rasa_nlu:
+      image: rasa/rasa_nlu:latest-spacy
+      volumes:
+        - ./models/rasa_nlu:/app/models
+      command:
+        - start
+        - --path
+        - models
+
+Cool, so we translated our ``docker run`` commands according to the
+docker-compose specification. While we it would be already to possible to
+run the containers in this configuration, one part is still missing: we have
+to instruct Rasa Core to connect to Rasa NLU to parse the user messages.
+To do so, we create a file called ``config/endpoints.yml``:
+
+.. code-block:: bash
+
+  mkdir config
+  touch config/endpoints.yml
+
+Docker containers which are started using Docker Compose are using the same
+network. Hence, each service can access other services by their service name.
+E.g., ``rasa_core`` can access ``rasa_nlu`` using ``rasa_nlu`` as domain name.
+Therefore, we put the following content in ``config/endpoints.yml``.
+
+.. code-block:: yaml
+
+  nlu:
+    url: http://rasa_nlu:5000
+
+Furthermore, we have to mount the ``config`` directory into the Rasa Core
+container. Also we need to instruct Rasa Core to use this configuration as
+well as specify the NLU model to use. These is done with the
+``--endpoints <path to endpoints.yml>`` parameter and
+``-u <nlu model to use>``. By adding these to our ``docker-compose.yml`` it
+should have the following content:
+
+.. code-block:: yaml
+
+  version: '3.0'
+
+  services:
+    rasa_core:
+      image: rasa/rasa_core:latest
+      ports:
+        - 5005:5005
+      volumes:
+        - ./models/rasa_core:/app/models
+        - ./config/:/app/config
+      command:
+        - start
+        - --core
+        - models
+        - -c
+        - rest
+        - --endpoints
+        - config/endpoints.yml
+        - -u
+        - current/
+    rasa_nlu:
+      image: rasa/rasa_nlu:latest-spacy
+      volumes:
+        - ./models/rasa_nlu:/app/models
+      command:
+        - start
+        - --path
+        - models
+
+Well done! We have got everything hooked up now and can finally talk to our
+bot!
+
+3.3 Speak to the Bot
+--------------------
+
+The REST API of Rasa Core is now available on ``http://localhost:5005``.
+To send messages to your bot, try:
+
+.. code-block:: bash
+
+  curl --request POST \
+    --url http://localhost:5005/webhooks/rest/webhook \
+    --header 'content-type: application/json' \
+    --data '{
+      "message": "hello"
+    }'
+
+Your bot should then answer something like:
+
+.. code-bock:: bash
+
+  [
+    {
+      "recipient_id": "default",
+      "text": "Hi, how is it going?"
+    }
+  ]
+
+If the bot cannot understand you (which is quite like since we provided only
+very little NLU training data), the answer is something along the lines:
+
+.. code-block:: bash
+
+  []
+
+Hooray! 🎉 🎊 We can now train Rasa Core, Rasa NLU and speak to our bot in
```

Comment:
Should this really be about training? I think the training steps are already included above.